internal/host/validator.go

```diff
@@ -1526,3 +1526,24 @@ func (v *validator) printVSphereUUIDEnabled(_ *validationContext, status Validat
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *validator) compatibleAgent(c *validationContext) ValidationStatus {
```

Comment:
i think new validation format was pushed already
@paul-maidment  ?

Reply:
Not merged yet - https://github.com/openshift/assisted-service/pull/4130